### PR TITLE
Episode 18, Part 2

### PR DESCRIPTION
**Pull request by sockeqwe**
Rss Feed [is valid](http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode18_part2/feed.rss ): ✅  

TODO:

- [x] Upload `The.Context.episode.18.part2.mp3` to your server (see google drive)
- [x] **Add link to Artem's auto setup** in `feed.rss` and `Episode_18_Part2.md`
- [ ]  Add and push git tag
- [ ]  Upload `.mp3` to githubs release section

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -1368,7 +1368,7 @@
 				<ul>
 				<li><a href="https://medium.com/exploring-android/android-tv-chill-3ba9c413daef">Android TV &amp; Chill</a>: A concept for what it would be like ordering fast-food on Android TV by Joe Birch.</li>
 				<li><a href="https://www.reddit.com/r/Android/comments/7zf4tk/openauto_turns_a_raspberry_pi_into_an_android/">OpenAuto</a>: Turns a raspberry pi into an Android Auto device</li>
-				<li><a href="TODO">Artem's auto setup</a></li>
+				<li><a href="https://www.amazon.com/dp/B074Y71ZKC?ref=yo_pop_ma_swf">Artem's car system running Android</a></li>
```
Comment (vanniktech) [inline feed.rss]: :car: :trollface:

File: show_notes/Episode_18_Part2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Episode 18, Part 2: Android Everywhere
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/92)
+
+In the second part of episode #18 Artur, Hannes and Artem discuss about Android TV, Chromecast and Android Auto.
```
Comment (ming13) [inline show_notes/Episode_18_Part2.md]: _discuss Android TV, Chromecast and Android Auto_

File: show_notes/Episode_18_Part2.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+#### Hosts:
+
+- Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](https://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+- Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+- Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](https://arturdryomov.online/), [GitHub](https://github.com/ming13)
+  
+
+#### Links:
```
Comment (ming13) [inline show_notes/Episode_18_Part2.md]: Empy line?

File: show_notes/Episode_18_Part2.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- [The Context Episode 5: Android TV with Joe Birch](https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_5.md)
+- [Android TV & Chill](https://medium.com/exploring-android/android-tv-chill-3ba9c413daef): A concept for what it would be like ordering fast-food on Android TV by Joe Birch.
+- [OpenAuto](https://www.reddit.com/r/Android/comments/7zf4tk/openauto_turns_a_raspberry_pi_into_an_android/): Turns a raspberry pi into an Android Auto device
```
Comment (ming13) [inline show_notes/Episode_18_Part2.md]: _Raspberry Pi_  ... _Android Auto device._

File: show_notes/Episode_18_Part2.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- [Artem's car system running Android](https://www.amazon.com/dp/B074Y71ZKC?ref=yo_pop_ma_swf)
```
Comment (ming13) [inline show_notes/Episode_18_Part2.md]: `https://www.amazon.com/dp/B074Y71ZKC`

File: show_notes/Episode_18_Part2.md (hunk 2 above)
Comment (sockeqwe) [reply show_notes/Episode_18_Part2.md]: What do you mean?

Comment (vanniktech) [reply show_notes/Episode_18_Part2.md]: No `about`

Comment (ming13) [reply show_notes/Episode_18_Part2.md]: Yep, thanks @vanniktech.

File: show_notes/Episode_18_Part2.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](https://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+- Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+- Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](https://arturdryomov.online/), [GitHub](https://github.com/ming13)
+  
+#### Links:
+
+- [The Context Episode 5: Android TV with Joe Birch](https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_5.md)
+- [Android TV & Chill](https://medium.com/exploring-android/android-tv-chill-3ba9c413daef): A concept for what it would be like ordering fast-food on Android TV by Joe Birch.
+- [OpenAuto](https://www.reddit.com/r/Android/comments/7zf4tk/openauto_turns_a_raspberry_pi_into_an_android/): Turns a Raspberry Pi into an Android Auto device
```
Comment (ming13) [inline show_notes/Episode_18_Part2.md]: Nit: a dot in the end.

**Comment by sockeqwe** [comment]
Released, thanks for reviewing @ming13  and @vanniktech

**Pull request merged**